### PR TITLE
Resources Screen

### PR DESCRIPTION
**Pull request by kennethlien**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Added the community resources screen! Click on the hamburger menu to navigate to it. It's still lacking styling and needs to implement base components, so that's work for next time.

UPDATE: It now seperates links into categories, added card component for resources.

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

This page is super simple, but most the stuff was built in the News screen PR.

## How to review

[//]: # "The order in which to review files and what to expect when testing locally"

Make sure the links work, and suggest styling changes.

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

Add style, use reusable components!

UPDATE: STYLE

## Tests Performed, Edge Cases

Clicking went well! Scrolling still works!

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @tommypoa 
@wangannie

### CODE REVIEW

File: screens/resources/ResourceScreen.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+class ResourceScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <View>
+        <TouchableOpacity
```
Comment (anniero98) [inline screens/resources/ResourceScreen.js]: You probably want to put this in at least a styles object, so that it's easily exported to a stylesheet and then can be refactored to `styled-components`. Since the Resources screen is going to be more complex than we thought, I think it's worth it to pull this out and add it to the `styled/` folder - you can make it `resources.js` (-:

File: styled/resources.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components/native';
+import { PoppinsText } from './shared';
```
Comment (anniero98) [inline styled/resources.js]: This is deprecated - import from `BaseComponents` if you want to extend for text

File: styled/resources.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+export const DateText = styled(PoppinsText)`
+  font-weight: bold;
+`;
+
+export const ContentContainer = styled.View`
+  flex-direction: column;
+  flex: 4;
+`;
+
+export const ContentText = styled(PoppinsText)`
```
Comment (anniero98) [inline styled/resources.js]: Let's get rid of all of these besides the `TopText`, since we aren't using any of those. Try and keep only what's being used right now !

File: screens/resources/ResourceScreen.js (hunk 1 above)
Comment (tommypoa) [reply screens/resources/ResourceScreen.js]: Given this week's Figma and our latest idea of the Resources page, I think it would be a good idea to create a new component - possibly something like `ResourceCard` - in addition to the `styled-components` in `resources.js`! The main reason is that all the resources links are in the format (style + content-wise).
